behaviours/common.go

```diff
@@ -0,0 +1,72 @@
+package behaviours
+
```

Comment:
move all the package to the root, we don't want to go importing a bazillion packages for very core functionality